packages/nextra-theme-blog/src/nav.tsx

```diff
@@ -28,9 +28,9 @@ export default function Nav({
             </Link>
           )
         })}
-        {config.navs?.map(nav => (
-          <Link key={nav.url} href={nav.url} passHref>
-            <a {...nav}>{nav.name}</a>
+        {config.navs?.map(({ href, children })  => (
+          <Link key={href} href={href} passHref>
+            <a>{children}</a>
```

Comment:
unneded spread, for example previously for `{ url: '/myurl', name: 'MYNAME' }` it's will be 
```jsx
<a href="/myurl" name="MYNAME">MYNAME</a>
```